includes/AJAX.php

```diff
@@ -165,6 +175,17 @@ public function import_products_from_square( $request = null ) {
 		wp_send_json_success( __( 'Your products are being imported in the background! This may take some time to complete.', 'woocommerce-square' ) );
 	}
 
+	/**
+	 * Update the connected_page_visited option.
+	 * This will be used to show a message to visit the onboarding
+	 * wizard and the onboarding wizard submenu until found `true`.
+	 *
+	 * @since 4.7.0
+	 */
+	public function update_connected_page_visited() {
+		update_option( 'wc_square_connected_page_visited', true );
```

Comment:
Should we set the autoload option on this to `false`? Or is this option always needed?

Reply:
I think autoload for `update_option` defaults to `null`, not sure if that works or not. Though I did pass `false`.